wandb/util.py

```diff
@@ -248,6 +252,9 @@ def json_friendly(obj):
 
     if is_tf_tensor_typename(typename):
         obj = obj.eval()
+    elif is_jax_tensor_typename(typename):
+        import numpy
```

Comment:
If we're sure it's jax we should `import jax.numpy as np` the user might not have numpy.

Reply:
This is the recommended way to move the tensor from the gpu to cpu in jax. Jax support will have to include numpy support.